sphinx/jinja2glue.py

```diff
@@ -208,13 +211,13 @@ def get_source(self, environment: Environment, template: str) -> tuple[str, str,
         loaders = self.loaders
         # exclamation mark starts search from theme
         if template.startswith('!'):
-            loaders = loaders[self.templatepathlen:]
+            loaders = loaders[self.templatepathlen :]
```

Comment:
This is definitely not a good idea I think. Would it be possible not to haveb those kind of changes? (also, would it mean that `x[1:]` would be formatted as `x[1: ]`? because if it's the case, then I think it's bad).

Reply:
As already mentioned, this is not changeable and completely PEP compliant: this is PEP compliant, and has already all been discussed in black/rust: https://github.com/astral-sh/ruff/pull/8654,
personally I think it is absolutely fine